<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_UserName">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>184</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>  UserName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>  Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>184</y>
      <width>38</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Login/login4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Password">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Login/Login5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Login">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Login/Login12.png);
font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>            Login  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Rain">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>-10</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Login/Login11.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Signup">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Forgotpassword">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Forgot Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SignIn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
